components/mailbox/src/index.html

```diff
@@ -67,21 +66,24 @@
           mailbox.sentMessageUpdate(message);
         });
 
-        const events = [
+        const mailboxEvents = [
           "replyClicked",
           "replyAllClicked",
           "forwardClicked",
-          "draftThreadEvent",
+          "draftThreadClicked",
+          "draftClicked",
         ];
-        events.forEach((eventType) =>
-          mailbox.addEventListener(eventType, (e) =>
-            toggleComposer(e, composer, eventType),
-          ),
+        mailboxEvents.forEach((eventType) =>
+          mailbox.addEventListener(eventType, (event) => {
+            console.log(eventType, { ...event.detail });
```

Comment:
✂️

Reply:
I thought we purposely kept these logging behaviours in this demo? Since these are the events user will need listen to and they can know if they are working.

All I did is just moved the `console.log` from the `toggleComposer` function above: [index.html](https://github.com/nylas/components/pull/417/files#diff-c581fcb635d197f2f77d850597322a79ed66aebfaec58a2b7e1f919e8ebde8a2#L22-L23)

Reply:
I actually left the `console.log` in `toggleComposer()` in by mistake 😂